esp8266/Makefile

```diff
@@ -148,7 +148,7 @@ DRIVERS_SRC_C = $(addprefix drivers/,\
 SRC_S = \
 	gchelper.s \
 
-FROZEN_MPY_PY_FILES := $(shell find -L $(FROZEN_MPY_DIR) -type f -name '*.py')
+FROZEN_MPY_PY_FILES := $(shell find -L $(FROZEN_MPY_DIR) -type f -name '*.py') modules/upip.py modules/upip_utarfile.py
```

Comment:
To get the dependencies right. First, at all, so upip was uncompressed, then at right time - uncompressing upip after *.mpy's were produced from modules/ doesn't help.

I assume all the above is enough to answer my question: superfluous complex automation is worse than superfluous simple duplication.